frontend/src/main-components/CreateCardContent.js

```diff
@@ -1,22 +1,131 @@
-import React from "react";
+import React, { Component } from "react";
 import styled from "@emotion/styled";
+import css from "./CreateCardContent.css";
+import { motion } from "framer-motion";
+import FrontCard from "../flashcards/FlashcardFrontPreview";
+import BackCard from "../flashcards/FlashcardBackPreview";
+import { getTranslation } from "../sub-components/translate";
+import LangaugeScroll from "../sub-components/languageScroll";
+class CreateCardContent extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      imgSrc: "",
+      text: "none",
+      translation: "none",
+      fromLang: "none",
+      toLang: "none",
+    };
+    this.TranslateText = this.TranslateText.bind(this);
+    this.fromLangSelected = this.fromLangSelected.bind(this);
+    this.toLangSelected = this.toLangSelected.bind(this);
+  }
+  /** 
+   * When the user has finished typing, the Google translate
+   * API is called to fetch the translated version of the text input.
+   */
+  TranslateText(event) {
+    event.preventDefault();
```

Comment:
Why do we need to prevent default?

Reply:
Actually I don't need to prevent the default here.